front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -58,11 +58,14 @@ export const formatSuggestedViasToRowVias = (
     const isMarginValid = theoreticalMargin ? marginRegExValidation.test(theoreticalMargin) : true;
     const departure =
       stopFor && arrival ? secToHoursString(time2sec(arrival) + Number(stopFor), true) : undefined;
+    const scheduledDepartureTime = startTime
```

Comment:
If I understand correctly, `scheduledDepartureTime` is used only for the departure.

I think it would be better to do smth like this: 
```
    let departure: string | undefined;
    if (stopFor) {
      if (i === 0) {
        departure = ....
      } else {
        departure = ...
      }
    }
```
This way, we have only 1 const `departure`